contracts/secret-storage/SecretManager.sol

```diff
@@ -763,6 +764,9 @@ contract SecretManager is
                 continue;
             address enclaveAddress = userStorage[_secretId].selectedEnclaves[index].enclaveAddress;
             uint256 lastAliveTimestamp = SECRET_STORE.getSecretStoreLastAliveTimestamp(enclaveAddress);
+            uint256 endTimestamp = userStorage[_secretId].endTimestamp;
```

Comment:
can move this to out of for loop?

Reply:
yeah, but since it's a view only function and not being used anywhere while state update, so won't make any difference.